examples/protobuff-tool/helloWorld.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+service helloWorld {
+		rpc hello(HelloRequest) returns (HelloResponse);
```

Comment:
Seems like these lines are not properly formatted.

Reply:
Fixed 86fd958